src/main/kotlin/me/joypri/Mix.kt

```diff
@@ -285,11 +285,15 @@ class RoleMixDelegate<V>(private val value: V) : ReadOnlyProperty<Mix, V> {
 
 data class Part constructor(val keyName: String, val value: Any)
 
+@Deprecated(
+    "Instead use `ThisRole has value` to be explicitly different from Kotlin's standard `to`.",
+    ReplaceWith("has"))
+infix fun <V : Any> Role<V>.to(value: V): Part = has(value)
+
 /**
- * TODO: Change this name to `has`?
  * TODO: Provide a counterpart for `MixRole`.
  */
-infix fun <V : Any> Role<V>.to(value: V): Part {
+infix fun <V : Any> Role<V>.has(value: V): Part {
```

Comment:
I understand why we don't want to use `to`, but I'm not a big fan of `has` it seems less "English" to say `FirstName has "Fred"`. What do you think of using `setTo` or `be`? While it sounds kinda funny, `FirstName be "Fred"` seems better to me than `has`.

Reply:
How about `of`?

~~~ Kotlin
val fred = Employee(
    FirstName of "Fred",
    Age of fredAge,
    TheirJob of Job.IC,
    TheirHrInfo of HrInfo(
        EmployeeNumber of fredEmployeeNumber,
        HireDate of fredHireDate
    )
)
~~~

Reply:
I like `of`. Let's go with that.